Update README.md (#11)
grafana/agent-modules: README.md

- <b>example</b>: A practical example shown for each module loader plus without modules for comparison
- <b>util</b>: Utilities for managing modules in this repo

## Modules

| Name |  Description | Agent Version | 
| ---- |  ----------- | ------------- | 
| [OTLP to LGTM](./modules/otlp/otlp-to-lgtm/) | Module to ingest OTLP data and then send it to Loki, Mimir and Tempo stacks locally or in GrafanaCloud. | `>= v0.33`
| [Grafana Agent Telemetry to LGTM](./modules/grafana-agent/telemetry-to-lgtm/) | Module to forward the Grafana Agent's own telemetry data to Loki, Mimir and Tempo stacks locally or in Grafana Cloud. | `>= v0.33`
| [Grafana Cloud Autoconfigure](./modules/grafana-cloud/autoconfigure/) | Module to automatically configure receivers for Grafana Cloud. | `>= v0.34`
| [Host Filtering](./modules/host-filter/) | The host filtering module provides a Flow mode equivalent to static mode's host filtering functionality. | `>= v0.34`

## Submitting modules

> Modules should be added under the `modules` folder. Each module should have a README.MD that has:

Reviewer: ```suggestion
Add modules to the `modules` folder. Each module must have a README.MD that provides the following information:
```

Be explicit and active in the instructions. "Should" implies the reader doesn't need to do the thing.... they can, but it's not required.

* Name
* Brief description
* Applicable Agent Versions
* Arguments
* Exports
* Example

> Module contents should be ordered:
> * `arguments`
> * `exports`
> * body of the module

Reviewer: ```suggestion
Modules must contain the following elements:
* Arguments
* Exports
* The body of the module

```

I removed the code markdown for arguments and exports since we are referring to them in general vs specifics. Order is important? Critical? Will things work if someone reverses the definitions for arguments and exports?

Do we provide a module content template for people to follow/fill in?

Author: Nothing will break, its a suggestion. We should create a template, I can work on that here in a few.

| ---- |  ----------- | ------------- | 
| [OTLP to LGTM](./modules/otlp/otlp-to-lgtm/) | Module to ingest OTLP data and then send it to Loki, Mimir and Tempo stacks locally or in GrafanaCloud. | `>= v0.33`
| [Grafana Agent Telemetry to LGTM](./modules/grafana-agent/telemetry-to-lgtm/) | Module to forward the Grafana Agent's own telemetry data to Loki, Mimir and Tempo stacks locally or in GrafanaCloud. | `>= v0.33`

## Submitting modules

> Modules should be added under `modules` folder. Each module should have a README.MD that has:

Reviewer 2: ```suggestion
Modules should be added under the `modules` folder. Each module should have a README.MD that has:
```

* Name
* Brief description
* Applicable Agent Versions
* Arguments
* Exports
* Example

> Module text should follow the order of `arguments`, `exports` then the body of the module. 

Reviewer 2: ```suggestion
Module contents should be ordered:
* `arguments`
* `exports`
* body of the module. 
```
